src/eip712/transaction.rs

```diff
@@ -29,6 +29,8 @@ pub struct Eip712Transaction {
     pub data: Bytes,
     pub factory_deps: Vec<Bytes>,
     pub paymaster_input: Bytes,
+    #[serde(skip_serializing)]
```

Comment:
Just curious about why / what is this line doing?

Reply:
It was an old patch for some endpoint related with the memory node but it seems that is already fixed so it's not necessary anymore. Good catch